### PR TITLE
Create product photo migration

### PR DESCRIPTION
**Pull request by RRicardotj**
This PR creates product photo migration required for next iteration of groups creation

### CODE REVIEW

File: src/schema/types/product.graphql [hunk 1]
```diff
@@ -5,6 +5,7 @@ type Product {
   description: String
   price: Float!
   marketPrice: Float!
+  productPhotos: [ProductPhoto]!
```
Comment (abelosorio) [inline src/schema/types/product.graphql:8]: Do we need to call this argument productPhotos? Why don't we call it just photos?

File: src/schema/types/product.graphql [hunk 2]
```diff
@@ -13,6 +14,7 @@ input ProductInput {
   description: String
   price: Float!
   marketPrice: Float!
+  productPhotos: [String!]
```
Comment (abelosorio) [inline src/schema/types/product.graphql:17]: Shouldn't this be productPhotosUrls?

Comment (RRicardotj) [reply src/schema/types/product.graphql:17]: But these strings are the base64 images. Basically it's just strings. Should I create a scalar type?

File: src/schema/types/product.graphql (hunk 1 above)
Comment (RRicardotj) [reply src/schema/types/product.graphql:8]: Yup, you're right

File: src/schema/types/product.graphql (hunk 2 above)
Comment (RRicardotj) [reply src/schema/types/product.graphql:17]: Oh! You mean the property name!

**Pull request merged**